jscomp/test/DictTests.res

```diff
@@ -0,0 +1,8 @@
+let someString = "hello"
+
+let createdDict = dict{
```

Comment:
Maybe also add a `Dict<int>` for testing?

Reply:
Sure, adding.